src/index.js

```diff
@@ -88,6 +88,9 @@ Object.assign(app.locals, {
     environmentBannerMessage: config.hostingEnvironment.environmentBannerMessage,
   },
   gaTrackingId: config.hostingEnvironment.gaTrackingId,
+  assets: {
```

Comment:
you also need to change the bit to get the assets url as you have done in the others